spec/system/staff/editing_incident_details_spec.rb

```diff
@@ -8,6 +8,7 @@
   let(:incident) { incident_in_divisions staff.divisions }
   it 'allows editing reason codes' do
     create :reason_code, identifier: 'A-1', description: 'Falling bananas'
+    create :supplementary_reason_code, identifier: 'a-8', description: 'Miscellaneous'
```

Comment:
Why can't you use the `identifier` and `description` defined from the factory?

Reply:
I can, just chose to modify as little of the existing test as possible

Reply:
Ah, got it.